flash_cards/models.py

```diff
@@ -1,3 +1,21 @@
 from django.db import models
+from django.db import models
+from django.contrib.auth.models import AbstractUser
+# from django.utils.text import slugify
```

Comment:
```suggestion
from django.utils import timezone
# from django.utils.text import slugify
```
We need this in order to pass the User's current time to the created_at category